lib/nazrin/document_client.rb

```diff
@@ -37,5 +39,40 @@ def delete_document(id)
         ].to_json,
         content_type: 'application/json')
     end
+
+    def batch(operations)
+      ActiveSupport::Deprecation.warn 'config.debug_mode is deprecated. Use config.mode = \'sandbox\' instead.' and return nil if Nazrin.config.debug_mode
+      return nil if Nazrin.config.mode == 'sandbox'
+
+      documents = operations.each_with_object([]) do |(type, tuple), arr|
+        case type.to_sym
+        when :add, :create
+          tuple.each do |id, field_data|
+            arr.push(
+              type: 'add',
+              id: id,
+              fields: field_data
+            )
+          end
+        when :delete, :destroy
+          tuple.map do |id|
```

Comment:
Gonna change this and the other two to `each` rather than `map` as `map` implies that the return value matters.